### PR TITLE
Skip Integration tests on pontoon commits.

### PR DESCRIPTION
**Pull request by b4handjr**
I gave this a try again, seems to work. Build times are down about half as I am skipping everything to do with the integration tests.

### CODE REVIEW

File: bin/circleci/install-test-dependencies.sh [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -ex
 
+IS_PONTOON=$(git show -s --format=%s | grep -q 'Pontoon:' && echo 'true' || echo '')
+if [[ $IS_PONTOON ]]; then
+  echo "Skipping Storybook deploy on Pontoon commit.";
```
Comment (fzzzy) [inline bin/circleci/install-test-dependencies.sh]: I think this was a copy paste error, it should say Skipping Integration Tests instead?

File: bin/circleci/test-firefox-dev.sh [hunk 2]
```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 set -ex
 
+# Skip builds for Pontoon commits
+IS_PONTOON=$(git show -s --format=%s | grep -q 'Pontoon:' && echo 'true' || echo '')
+if [[ $IS_PONTOON ]]; then
+  echo "Skipping Storybook deploy on Pontoon commit.";
```
Comment (fzzzy) [inline bin/circleci/test-firefox-dev.sh]: Same here.

**Review by fzzzy — APPROVED** [review]
Looks good with nits. Please squash.

**Review by lmorchard — APPROVED** [review]
LGTM too

**Pull request merged**